src/System.Linq.Parallel/src/System/Linq/Parallel/Enumerables/AggregationMinMaxHelpers.cs

```diff
@@ -40,18 +42,20 @@ private static T Reduce(IEnumerable<T> source, int sign)
         // Helper method to find the minimum element in the source.
         //
 
+        [return: MaybeNull]
         internal static T ReduceMin(IEnumerable<T> source)
         {
-            return Reduce(source, -1);
+            return Reduce(source, -1)!;
         }
 
         //-----------------------------------------------------------------------------------
         // Helper method to find the maximum element in the source.
         //
 
+        [return: MaybeNull]
         internal static T ReduceMax(IEnumerable<T> source)
         {
-            return Reduce(source, 1);
+            return Reduce(source, 1)!;
```

Comment:
Same question.

Reply:
Actually compiler not complaining from command line, seems only from VS compiler, will remove bangs